src/client/pages/OfferNew/utils.ts

```diff
@@ -61,6 +61,17 @@ export const getOfferData = (quoteBundle: QuoteBundle): OfferData => {
   }
 }
 
+const getHouseholdSizeFromBundledQuotes = (quotes: BundledQuote[]): number => {
+  const quoteDetails = quotes.map((x) => x.quoteDetails)
```

Comment:
We usually don't use variable names like `x`, or `i` but try to generally be more explicit.
```suggestion
  const quoteDetails = quotes.map((quote) => quote.quoteDetails)
```